pdf_annotate/graphics.py

```diff
@@ -70,135 +78,244 @@ def join(stream1, stream2):
         return ContentStream(stream1.commands + stream2.commands)
 
 
-class NoOpTransformBase(object):
+@total_ordering
```

Comment:
Why are you using `@total_ordering` if you're just raising an error in `__lt__`? You're not defining comparisons in any other subclasses.

Reply:
`@total_ordering` propagates the error into `__gt__`, `__gte__`, etc. I wanted the PDF command classes to be un-orderable, so you can't sort them etc. This prevents anyone relying on the tuple ordering that many of these classes have.

Reply:
Ohhh, so you're using this to force them to _not_ be orderable. Sneaky.